.github/workflows/test.yml

```diff
@@ -9,30 +9,31 @@ jobs:
   test:
     name: Run NPM Test (e2e)
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [13.x]
-        mongodb-version: [4.2]
+    container: node:12-alpine
+    services:
+      postgres:
+        image: postgres:13-alpine
+        env:
+          POSTGRES_DB: db_test
+          POSTGRES_USER: admin
+          POSTGRES_PASSWORD: admin
+        # Set health checks to wait until postgres has started
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
-      - name: Use Node.js ${{ matrix.node-version }}
-        uses: actions/setup-node@v1
-        with:
-          node-version: ${{ matrix.node-version }}
-  
-      - name: Start MongoDB
-        uses: supercharge/mongodb-github-action@1.3.0
-        with:
-          mongodb-version: ${{ matrix.mongodb-version }}
-
       - name: NPM Test
         working-directory: ./server
         env:
-          MONGODB_URI: mongodb://localhost:27017/test
+          POSTGRES_DB: db_test
+          POSTGRES_USER: admin
+          POSTGRES_PASSWORD: admin
```

Comment:
I can't shake the dirty feeling of having this in source, even if it's intentionally a dummy value 😓 

I can't think of a solution more elegant than adding the test user/password to our .env though. Thoughts?

Reply:
But for tests on GH actions the dummy password will be exposed anyway, even using in an .env file. Should we use it as a secret?

Reply:
Discussed offline - since this is ephemeral and can be seen via the GH Actions, we aren't worried about exposing the default credentials here since they're a known entity for a PostgreSQL image anyway.